packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -219,7 +219,8 @@ function getTabbableCandidates(container: HTMLElement) {
   const walker = document.createTreeWalker(container, NodeFilter.SHOW_ELEMENT, {
     acceptNode: (node: any) => {
       const isHiddenInput = node.tagName === 'INPUT' && node.type === 'hidden';
-      if (node.disabled || node.hidden || isHiddenInput) return NodeFilter.FILTER_SKIP;
+      const isAnchor = node.tagName === 'A';
```

Comment:
I've had a chat with @jjenzz about this yesterday and we agreed to go ahead with the idea of not auto focusing links on mount as it probably is a better default for the 90% to skip links.

That said, I believe this is the wrong place for the checks as the `getTabbableCandidates` gets used for different purposes like finding out edges to loop focus around (in which case links should be included).

 It looks like it would rather need to be done precisely where we handle the autofocus on mount event, something like this (obviously the actual solution can be anything):

```diff
if (!mountEvent.defaultPrevented) {
-  focusFirst(getTabbableCandidates(container), { select: true });
+  focusFirst(removeLinks(getTabbableCandidates(container)), { select: true });
  if (document.activeElement === previouslyFocusedElement) {
    focus(container);
  }
}
```